### DISCUSSION TITLE
제페토 월드 영상 소스 외부 URL 설정시 심사 이슈

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#171] [5. Q&A]
안녕하세요.

제페토 월드 내에서 외부 URL 소스의 동영상을 재생하려고 하는데,
심사 통과가 가능할까요?
제페토 월드 가이드에는 **2.5.2. 월드 내 영상이 포함된 경우 승인 불가** 라고 나와있는데,
외부 URL 영상 뿐만 아니라 내부에 embed된 영상도 반려 사유가 되나요?
가이드 부탁드립니다.

감사합니다.

**Comment by beanyh** [comment]
네~ 내부에 embed된 영상도 반려 사유입니다. 
현재는 심사 운영상 동영상은 허용하고 있지 않는 점 이해 부탁드립니다.
